husky_control/config/control.yaml

```diff
@@ -23,7 +23,7 @@ husky_velocity_controller:
 
   # Wheel separation and radius multipliers
   wheel_separation_multiplier: 1.875 # default: 1.0
-  wheel_radius_multiplier    : 1.0 # default: 1.0
+  wheel_radius_multiplier    : 0.5 # default: 1.0
```

Comment:
No. But it is required to be 0.5 for the gear box to work with the dual_ur5_husky setup. Thus, before this PR can be merged we need to change the control.yaml include so that it is different between setups and parametrized in the launch file.